source/github-helpers/timeline-item.tsx

```diff
@@ -1,6 +1,11 @@
 import React from 'dom-chef';
 
-export default function TimelineItem(): JSX.Element {
+export function TimelineItemOld(): JSX.Element {
 	// Classes copied from #issuecomment-new + mt-3 added
 	return <div className="ml-0 pl-0 ml-md-6 pl-md-3 mt-3" />;
 }
+
+// https://github.com/refined-github/refined-github/pull/8141
+export function TimelineItem(): JSX.Element {
+	return <div className="my-2" />;
```

Comment:
It seems it only need `my-2` in new view. It works on all widths.

https://github.com/user-attachments/assets/976d2db5-95cb-489b-8901-a875206e9c7a